lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -6848,11 +6848,12 @@ Container tag set for the ``<xref>`` tags.
 A tool can refer multiple reference IDs.
 	
 ```xml
-   <!-- Example: this tool is seqtk -->
+   <!-- Example: this tool is dada2 -->
    <xrefs>
-	<xref type="bio.tools">seqtk</xref>
+       <xref type="bio.tools">dada2</xref>
+       <xref type="bioconductor">10.18129/B9.bioc.dada2</xref>
```

Comment:
It seems like all the DOIs are unversioned and start the same way - should we just have the xref be "dada2" and generate links from there?

Reply:
xref https://support.bioconductor.org/p/99617/

Reply:
I thought about that, but I'm not sure if this stays forever? Maybe it's not that much a risk that we take here? But maybe explicitly is better than implicitly here? Also if others reuse this and parse the tools?

@csoneson any idea?

Reply:
```diff
diff --git a/client/src/components/xrefs.vue b/client/src/components/xrefs.vue
index 0cdf65c2a7..dedaadaa7a 100644
--- a/client/src/components/xrefs.vue
+++ b/client/src/components/xrefs.vue
@@ -10,6 +10,9 @@
                     <template v-if="xref.reftype == 'bio.tools'">
                         <a :href="`https://bio.tools/${xref.value}`" target="_blank">{{ xref.value }}</a>
                     </template>
+                    <template v-if="xref.reftype == 'bioconductor'">
+                        <a :href="`https://www.bioconductor.org/packages/release/bioc/html/${xref.value}.html`" target="_blank">{{ xref.value }} (doi:10.18129/B9.bioc.{{ xref.value }})</a>
+                    </template>
                     <template v-else>
                         {{ xref.value }}
                     </template>
diff --git a/test/functional/tools/mulled_example_explicit.xml b/test/functional/tools/mulled_example_explicit.xml
index 03db8e436e..edbf26117d 100644
--- a/test/functional/tools/mulled_example_explicit.xml
+++ b/test/functional/tools/mulled_example_explicit.xml
@@ -16,6 +16,9 @@ bwa > $output_1 2>&1
     </outputs>
     <tests>
     </tests>
+    <xrefs>
+        <xref type="bioconductor">dada2</xref>
+    </xrefs>
     <help><![CDATA[
 TODO: Fill in help.
     ]]></help>
```

It will be easy enough to tell if these are full DOIs later on - so we might as well start simple IMO.

Reply:
I think the plan is that the DOI should stay the same form in the future as well, so I think it's pretty safe to generate them like this. When explicitly specifying the link target, the safest would be to link to https://bioconductor.org/packages/<pkg>/ rather than to the release landing page - this link should automatically point to the "right" version of a package (release if possible, devel if not yet in release, list of removed packages if deprecated).

Reply:
Thanks @csoneson, I made the URL change!